resource_management_mobile/src/app/auth/models/auth.model.ts

```diff
@@ -0,0 +1,11 @@
+export interface LoginRequest{
+    userName : string;
+    password : string;
+}
+
+export interface LoginResponse{
+    data :any;
```

Comment:
Don't use any - If you are adding "any" then add a comment to update in feature 

\** TODO: Have to update this property based on API response **\

Reply:
Removed.